internal/bminventory/inventory_test.go

```diff
@@ -123,6 +124,20 @@ func mockAMSSubscription(ctx context.Context) {
 	mockAccountsMgmt.EXPECT().CreateSubscription(ctx, gomock.Any(), gomock.Any()).Return(&amgmtv1.Subscription{}, nil)
 }
 
+func verifyUsageSet(db *gorm.DB, clusterId strfmt.UUID, usage models.Usage) {
```

Comment:
i don't think that this check belongs here, you are testing the `usage` package in unit tests of a different package.

Reply:
it will requires duplication of many updateCluster tests in another package. One can argue that reporting the usage is an integral part of the updating process

Reply:
it belongs to subsystem then, it's not unit test anymore.